tests/plots/test_declarative.py

```diff
@@ -1184,3 +1184,22 @@ def test_panel():
 
     pc.panel = panel
     assert pc.panel is panel
+
+
+def test_copy():
```

Comment:
Since this test function uses `MapPanel`, it *needs* cartopy, so we just need to note it as such.
```suggestion
@needs_cartopy
def test_copy():
```